django_oikotie/housing_company.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import dataclass
+
+from .utils import object_to_etree
```

Comment:
There are mostly absolute imports across the project, so I would like to stay uniform, so:

`from django_oikotie.utils import object_to_etree`